.github/workflows/test.yml

```diff
@@ -67,34 +72,35 @@ jobs:
         run: |
           pip cache remove llama_cpp_python
 
-      - name: Cache huggingface
-        uses: actions/cache@0c45773b623bea8c8e75f6c82b208c3cf94ea4f9 # v4.0.2
-        with:
-          path: ~/.cache/huggingface
-          # config contains DEFAULT_MODEL
-          key: huggingface-${{ hashFiles('src/instructlab/configuration.py') }}
-
-      - name: Install instructlab and instructlab-eval
+      - name: Start inference server
         run: |
           export PATH="/home/runner/.local/bin:/usr/local/cuda/bin:$PATH"
-          python3 -m venv venv
-          . venv/bin/activate
           cd instructlab
+          python3.11 -m venv cli_venv
+          . cli_venv/bin/activate
           sed 's/\[.*\]//' requirements.txt > constraints.txt
-          python3 -m pip cache remove llama_cpp_python
+          python3.11 -m pip cache remove llama_cpp_python
           CMAKE_ARGS="-DLLAMA_CUBLAS=on" python3 -m pip install --no-binary llama_cpp_python -c constraints.txt llama_cpp_python
           # needed for --4-bit-quant option to ilab train
-          python3 -m pip install bitsandbytes
+          python3.11 -m pip install bitsandbytes
           # install instructlab
-          python3 -m pip install .
-          cd ..
-          # Install instructlab-eval
-          python3 -m pip install .
+          python3.11 -m pip install .
+          # start llama-cpp server
+          ilab model download --repository instructlab/granite-7b-lab-GGUF --filename granite-7b-lab-Q4_K_M.gguf
```

Comment:
Since what we want to do with `ilab` is pretty basic, I'd suggest we just install instructlab from pypi.

Reply:
I would agree except the current pypi package is pretty out-of-date - using it now is just going to require us to change a bunch of stuff when the next release comes out - but once `0.18.0` lands that'll make sense

Reply:
I agree we should install from pypi, installing instructlab from pypi won't change the ilab model download command

Reply:
@nathan-weinberg we can install the beta releases with `--pre` and then remove that once 0.18.0 lands